src/components/Explore/ExploreChart.tsx

```diff
@@ -29,71 +85,115 @@ const ExploreChart: FunctionComponent<{
-        <RectClipGroup width={innerWidth} height={innerHeight}>
-          {series.map((serie, i) => (
-            <SeriesChart
-              key={`series-chart-${i}`}
-              data={serie.data}
-              x={d => timeScale(getDate(d)) || 0}
-              y={d => yScale(getY(d))}
-              type={serie.type}
+    <Styles.PositionRelative>
+      <svg width={width} height={height}>
+        <Group key="chart-container" top={marginTop} left={marginLeft}>
+          <Styles.GridLines>
+            <GridColumns<Date> scale={dateScale} height={innerHeight} />
+            <GridRows<number> scale={yScale} width={innerWidth} />
+          </Styles.GridLines>
+          <RectClipGroup width={innerWidth} height={innerHeight}>
+            {series.map(serie => (
+              <SeriesChart
+                key={`series-chart-${serie.label}`}
+                data={serie.data}
+                x={getXPosition}
+                y={getYPosition}
+                type={serie.type}
+                yMax={innerHeight}
+                barWidth={barWidth}
+              />
+            ))}
+          </RectClipGroup>
+          {tooltipOpen && tooltipData && (
+            <DataMarkers
+              x={getXPosition}
+              y={getYPosition}
               yMax={innerHeight}
               barWidth={barWidth}
+              series={series}
+              date={tooltipData.date}
             />
-          ))}
-        </RectClipGroup>
-      </Group>
-      <Group key="axes" top={marginTop} left={marginLeft}>
-        <Styles.GridLines>
-          <GridColumns<Date> scale={timeScale} height={innerHeight} />
-          <GridRows<number> scale={yScale} width={innerWidth} />
-        </Styles.GridLines>
-        <ChartStyle.Axis>
-          <AxisLeft scale={yScale} />
-        </ChartStyle.Axis>
-        <ChartStyle.Axis>
-          <AxisBottom
-            top={innerHeight}
-            scale={timeScale}
-            tickValues={timeTicks}
-            tickFormat={(date: Date) => moment(date).format(timeTickFormat)}
+          )}
+          <ChartOverlay
+            width={innerWidth}
+            height={innerHeight}
+            xScale={dateScale}
+            onMouseOver={onMouseOver}
+            onMouseLeave={hideTooltip}
           />
-        </ChartStyle.Axis>
-      </Group>
-    </svg>
+          <ChartStyle.Axis>
+            <AxisLeft scale={yScale} />
+            <AxisBottom
+              top={innerHeight}
+              scale={dateScale}
+              tickValues={timeTicks}
+              tickFormat={xTickFormat}
+            />
+          </ChartStyle.Axis>
+        </Group>
+      </svg>
+      {tooltipOpen && tooltipData && (
+        <ExploreTooltip
+          left={p => getXPosition(p) + marginLeft}
+          top={p => getYPosition(p) + marginTop}
+          date={tooltipData.date}
+          series={series}
+        />
+      )}
+      {tooltipOpen && tooltipData && (
```

Comment:
FWIW- We repeat this ~3 times...

Maybe we should have a:

```
const openToolTipDate = tooltipOpen && tooltipData && toolTipData.date;
```

helper variable or something?